### ISSUE TITLE
Any file named "CmdProxy.elm" is ignored!

### ISSUE THREAD

**Issue by tylerperkins** [klazuka/intellij-elm#540]
Whether I create a new Elm file called "CmdProxy.elm", or I rename an existing Elm file to that name, it displays as a plain, unformatted text file. It doesn't matter whether the file is in `src/` or in a subdirectory there. The contents of the file have no effect, even when `elm make ...` from the command line compiles it just fine. I've tried **Invalidate Caches / Restart ...** as well as reinstalling the Elm plugin, all efforts to no avail.

The only workaround I've found is to rename the file to _any_ other `*.elm` name. Then instantly, formatting, highlighting of errors, tools, etc. all suddenly work great!

Weird, huh?!

**Comment by klazuka** [comment]
I wasn't able to reproduce this. Can you create an example project that shows the problem?

Are you using IntelliJ, WebStorm, something else?

**Comment by tylerperkins** [comment]
OK, here's a zipped, minimal Elm project that exhibits the problem. I'm running **IntelliJ IDEA 2019.2.3 (Community Edition)**.
[CmdProxyBug.zip](https://github.com/klazuka/intellij-elm/files/3757395/CmdProxyBug.zip)

I should mention other circumstances, which might be relevant: I originally created file `CmdProxy.elm` using the contextual menu **New --> File**, as opposed to **New --> Elm File**.  The file displayed as plain text, even with the `.elm` extension. I realized I should have used **Elm File**, so I deleted the file and used **New --> Elm File** with the same filename. But it still displayed as plain text, unless I renamed it as described above. Now, the problem persists, even in a brand new project in which I correctly used **New --> Elm File**. Yet, I repeated this procedure with **AnotherArbitraryFile.elm**, and it displayed correctly with Elm highlighting.

**Comment by klazuka** [comment]
Hmm. Thanks for the example, but I can't download the file via that link (the hostname does not resolve).

Without having looked at your example, I suspect that IntelliJ has persisted something to your `.idea` directory that is causing the problem.

A simple, safe thing to do would be to try renaming `.idea` to something else. Then re-launch IntelliJ and see if you still have the problem. Then rename it back to `.idea` when you're done trying things out. 

For simple projects it's ok to `rm -rf .idea`, but if you have any saved run configurations, database connections, etc, you will lose those, so _don't do it_ if you have anything like that.

**Comment by tylerperkins** [comment]
Hi, Keith. You wrote:

> I can't download the file via that link (the hostname does not resolve).

Maybe you were offline when you tried? It's just a file hosted in your own Github account!: [https://github.com/klazuka/intellij-elm/files/3757395/CmdProxyBug.zip](https://github.com/klazuka/intellij-elm/files/3757395/CmdProxyBug.zip)

Anyway, this bug is quite bizarre, but I guess it's unlikely to effect anyone else -- nor even me. I've just resorted to using a name other than `CmdProxy.elm` for my file. In fact, I tried IntelliJ IDEA _Ultimate_ 2019.2.3, and indeed, the problem goes away.

It nags me, though. I've tried everything I can think of, including removing directories `.idea`, `~/Library/Application Support/IdeaIC2019.2/`, and `~/Library/Preferences/IntelliJIdea2019.2`, then even reinstalling IntellJ IDEA CE. The same bug survives. For being so weird, it's quite robust!

**Comment by Herteby** [comment]
@tylerperkins Have you tried running the `Invalidate Caches / Restart...` command?

**Comment by tylerperkins** [comment]
Yes, as I stated originally, that was the first thing I tried. To make sure, I just now tried it again, with the same result.
